src/api-token-resolver.ts

```diff
@@ -2,14 +2,17 @@ import * as rp from 'request-promise-native';
 
 export class ApiTokenResolver {
 
-  public static getAccessToken(host: string, apiToken: string) {
+  public static getAccessToken(apiToken: string) {
```

Comment:
That does not work, as other customers have different svc instances. Now the IP Whitelist topic is fixed, therefore please revert this to the instance name